daktari/checks/direnv.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, required_version: Optional[str] = None, recommended_version:
         self.recommended_version = recommended_version
         self.suggestions = {
             OS.GENERIC: "Install direnv: https://direnv.net/#getting-started",
+            OS.OS_X: "Install direnv using brew: <cmd>brew install direnv</cmd>",
+            OS.OS_X: "Install direnv using apt-get: <cmd>sudo apt-get install direnv</cmd>",
```

Comment:
This should be `OS.UBUNTU` I think?

Reply:
Kinda sad we don't get a type error or something from specifying the same key twice